client/CS130_Team13/Assets/Scripts/RemoteController.cs

```diff
@@ -4,6 +4,20 @@
 using UnityEngine.Networking;
 using System.Collections;
 
+/////// USE THIS FOR CLIENT COMMUNICATION FOR NOW :)
+public class RemoteControllerTEMP {
```

Comment:
call this one RemoteController, and rename the test version to RemoteControllerTEMP